tests/test_mesh_periodic.py

```diff
@@ -175,7 +175,7 @@ def octet_truss_heterogeneous(rve_unit: Rve) -> (cq.Compound, List[Phase]):
     ],
 )
 def test_octettruss_mesh_must_be_periodic(
-    shape: Union[cq.Compound, cq.Shape],
+    shape: Tuple[Union[cq.Compound, cq.Shape], List[Phase], Rve],
     request,

```

Comment:
`request` parameter should be typed as `FixtureRequest`

`from pytest import FixtureRequest`